### ISSUE TITLE
Add date published & date next updated expected to Vacancy data

### ISSUE THREAD

**Issue by ODILMichelle** [open-innovations/yff-data#167]
Format to look like that seen on labour market page: https://data.youthfuturesfoundation.org/dashboard/labour-market/

(No JSon file that I could see to pull this from)

**Comment by ODILMichelle** [comment]
@taz-chiles is this something a) that we can handle without Giles and b) can add to Backlog? Or do we need another ticket set up for creating this JSon file?

**Comment by taz-chiles** [comment]
@ODILMichelle I have added the latest monthly and quarterly release dates to the page content. I have also updated the page content to refer to the specific datasets rather than the analysis publication for which the link will likely change. 

@gilesdring I cannot see in the repo where you have pulled the upcoming release date from, could you point me in the right direction?

**Comment by ODILMichelle** [comment]
What does done look like?
* Add dates published & next publication date to be within each heading, rather than 'About this Data'
* Has 'next publication date'..

**Comment by taz-chiles** [comment]
I have added the last publication date for both datasets (they are the same because they are from the same LMS release). 

After speaking with Christian about pulling the data from EDD, it will be possible to pull the upcoming date soon but this isn't possible currently. Given I have already started re-writing the pipelines to pull data from EDD, I think we should close this ticket and open another to add a future release date soon -

@ODILMichelle thoughts?
